app/presenters/report_presenter.rb

```diff
@@ -35,7 +35,10 @@ def due_defects_by_priority(priority:)
 
   def overdue_defects_by_priority(priority:)
```

Comment:
Does this method name still reflect the behaviour?

Reply:
I believe so - as I understand it, an overdue defect is:
- One closed late in the past (i.e. was overdue in the past)
- One that's still open past its target completion date
- One that's closed without a completion date (I suppose this is the most controversial addition, but there shouldn't be many of these and it's to encourage Hackney to add a date to these manually in the case where it's missing)

Reply:
I guess what I mean is, it might be what we display as overdue, but if someone wanted to do something else with overdue defects, would this give them what they expected. Don't let it block you, though.